src/index.js

```diff
@@ -1 +1,18 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+import UserScore from './modules/usersScore.js';
+
+const newUserScore = new UserScore();
+const form = document.querySelector('.form');
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const user = form.name.value;
+  const score = form.score.value;
+  newUserScore.addNewScore({ user, score });
+  form.name.value = '';
+  form.score.value = '';
+});
+
+const refersh = document.getElementById('refresh');
+refersh.addEventListener('click', newUserScore.fetchData);
```

Comment:
- [ ] Please fix the double render of your data as the last reviewer mentioned.
You can do it by passing a custom function to refresh the button event listener and in it you can reset the data in the dom before you call `fetchData`.

Reply:
Hi @Omar-Muhamad, I tried this but it's still not working 
`refersh.addEventListener('click', () => {
    document.getElementById('scores-container').innerHTML = "";
    return newUserScore.fetchData;
});`

Reply:
Hello @Omar-Muhamad,
I have solved the problem.
Thank you for the code review.